test/Reddit.spec.ts

```diff
@@ -2,400 +2,478 @@ import * as utils from "../scripts/helpers/utils";
-        const accountRoot = await IMTInstance.getTreeRoot();
-        const userPDAProof = await pubkeyStore.getPDAMerkleProof(User.Path);
-
-
-        const resultProcessTx = await rollupRedditInstance.processBurnConsentTx(
-            balanceRoot,
-            accountRoot,
-            tx.signature,
-            txBytes,
-            userPDAProof,
-            userMP
-        )
-        const [newBalanceRoot, errorCode] = [resultProcessTx[0], resultProcessTx[2]];
-        assert.equal(errorCode, ErrorCode.NoError);
-        assert.equal(newBalanceRoot, result[1]);
-
-        const compressedTx = await RollupUtilsInstance.CompressBurnConsentNoStruct(
-            tx.fromIndex, tx.amount, tx.nonce, tx.cancel, tx.signature
-        );
-        await RollupUtilsInstance.DecompressBurnConsent(compressedTx);
-
-        await rollupCoreInstance.submitBatch(
-            [compressedTx],
-            newBalanceRoot,
-            Usage.BurnConsent,
-            { value: ethers.utils.parseEther("32").toString() }
-        );
-
-        assert.equal(newBalanceRoot, await accountStore.getRoot());
-
-    })
-    it("lets Reddit to execute the burn", async function () {
-        const userMP = await accountStore.getAccountMerkleProof(User.AccID);
-        const tx = {
-            fromIndex: User.AccID,
-        } as BurnExecutionTx
-        const signBytes = await RollupUtilsInstance.BurnExecutionSignBytes(
-            tx.fromIndex
-        );
-        tx.signature = utils.sign(signBytes, User.Wallet);
-        const txBytes = await RollupUtilsInstance.BytesFromBurnExecutionNoStruct(
-            tx.fromIndex
-        );
-        await RollupUtilsInstance.BurnExecutionFromBytes(txBytes);
-
-
-        const result = await rollupRedditInstance.ApplyBurnExecutionTx(userMP, txBytes);
-        const userUpdatedAccount = await utils.AccountFromBytes(result[0]);
-        await accountStore.update(User.AccID, userUpdatedAccount);
-
-        const balanceRoot = await rollupCoreInstance.getLatestBalanceTreeRoot();
-
-        const resultProcessTx = await rollupRedditInstance.processBurnExecutionTx(
-            balanceRoot,
-            txBytes,
-            userMP
-        )
-        const [newBalanceRoot, errorCode] = [resultProcessTx[0], resultProcessTx[2]];
-        assert.equal(errorCode, ErrorCode.NoError, "processTx returns error");
-        assert.equal(newBalanceRoot, result[1], "mismatch balance root");
-
-        const compressedTx = await RollupUtilsInstance.CompressBurnExecutionNoStruct(
-            tx.fromIndex, tx.signature
-        );
-        await RollupUtilsInstance.DecompressBurnExecution(compressedTx);
-
-        await rollupCoreInstance.submitBatch(
-            [compressedTx],
-            newBalanceRoot,
-            Usage.BurnExecution,
-            { value: ethers.utils.parseEther("32").toString() }
-        );
-
-        assert.equal(newBalanceRoot, await accountStore.getRoot());
-
-    })
-})
+  let wallets: any;
```

Comment:
Could we avoid breaking the style in such a big diff? We could have another PR to unify the style of the whole codebase.

Reply:
Ah! yeah, sorry about that.